spec/game_statistics_spec.rb

```diff
@@ -45,7 +45,19 @@
 
   describe "#percentage_home_wins" do
     it "returns a percentage of games that a home team has won as a float (rounded to the nearest 100th)" do
-      # expect(@game_stats.percentage_home_wins).to be_a(Float)
+      expect(@game_stats.percentage_home_wins).to be_a(Float)
+    end
+  end
+
+  describe "#percentage_visitor_wins" do
+    it "returns a percentage of games that a visiting team has won as a float (rounded to the nearest 100th)" do
+      expect(@game_stats.percentage_visitor_wins).to be_a(Float)
+    end
+  end
+
+    describe "#percentage_ties" do
+      it "returns a percentage of games that both teams have won as a float (rounded to the nearest 100th)" do
+        expect(@game_stats.percentage_home_wins).to be_a(Float)
```

Comment:
we should also make an assertion here that:
 expect(@game_stats.percentage_ties).to eq(0.20)

Reply:
Updated and corrected line 60 expect statement